app/templates/components/public/session-item.hbs

```diff
@@ -0,0 +1,52 @@
+{{#ui-accordion}}
+  <div class="title">
+    <div class="ui">
+      <h3 class="ui header">
+        {{session.title}}
+        <div class="sub header">
+          {{session.sessionType.name}}
+        </div>
+      </h3>
+      <div class="ui row">Talk</div>
+    </div>
+    <br>
+    <div class="ui grid">
+      <div class="column twelve wide">
+        <div class="session-speakers">
+          {{#each session.speakers as |speaker|}}
+            <img alt="speaker" class="ui mini avatar image" src="{{if speaker.photo.iconImageUrl speaker.image '/images/placeholders/avatar.png'}}">
+          {{/each}}
+        </div>
+      </div>
+      <div class="ui right aligned column four wide" style="padding-left: 0">
+        <div class="">{{session.microlocation.name}} <i class="marker icon"></i></div>
+        <div class="">{{moment-format session.startAt 'hh:mm a / DD-MM-YYYY'}} <i class="wait icon"></i></div>
+      </div>
+    </div>
+    <div class="ui row">
```

Comment:
@geekyd

Looks like the row is out side the grid ? 

Should be something like this.

```hbs
<div class="ui grid">
    <div class="row">
        <!-- speaker pics and microlocation/time -->
    </div>
<div class="row">
    <div class="column">
        {{session.shortAbstract}}
    </div>
</div>
```

A `row` cannot exist outside a `ui grid`